explore/src/main/scala/explore/targeteditor/AladinContainer.scala

```diff
@@ -29,7 +29,7 @@ import react.common._
 import scala.annotation.nowarn
 import scala.concurrent.duration._
 
-final case class AladinContainer(
+final case class AladinContainer( // TODO Use target's FOV if not set in user prefs.
```

Comment:
Depending on the target model we choose, saving the target FOV in user prefs will be more or less difficult. Currently in staging, I don't think it really works with the unshared TargetEnv (although I'd have to look). Maybe we could use target name? With an unshared model, using target id has limitations.

Reply:
Understood.